src/test/java/org/prebid/server/auction/BidderAliasesTest.java

```diff
@@ -0,0 +1,189 @@
+package org.prebid.server.auction;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.prebid.server.bidder.BidderCatalog;
+
+import static java.util.Collections.singletonMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+public class BidderAliasesTest {
+
+    @Rule
+    public final MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private BidderCatalog bidderCatalog;
+
+    @Before
+    public void setUp() {
+        given(bidderCatalog.isActive(anyString())).willReturn(true);
+    }
+
+    @Test
+    public void isAliasDefinedShouldQueryCatalogWhenNoAliasesInRequest() {
+        // given
+        given(bidderCatalog.isAlias(anyString())).willReturn(true);
+
+        final BidderAliases aliases = BidderAliases.of(bidderCatalog);
```

Comment:
Minor. Consider moving `aliases` as class field with default initialization in `setUp()`.